### PR TITLE
[IMP] mail: email template previews attendee desc order

### PR DESCRIPTION
**Pull request by ppr-odoo**
**Task:**
https://www.odoo.com/web#id=34182&view_type=form&model=project.task&action=333&active_id=131&menu_id=4720
**Pad:**
https://pad.odoo.com/p/r.bfc1444a6731258c55ab958dff25820d

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/mail/wizard/email_template_preview.py [hunk 1]
```diff
@@ -17,7 +17,7 @@ def _get_records(self):
         if not template_id:
             return []
         template = self.env['mail.template'].browse(int(template_id))
-        records = self.env[template.model_id.model].search([], limit=10)
+        records = self.env[template.model_id.model].search([], order="id desc", limit=10)
```
Comment (rim-odoo) [inline addons/mail/wizard/email_template_preview.py]: ok

File: addons/mail/wizard/email_template_preview.py [hunk 2]
```diff
@@ -38,8 +38,10 @@ def default_get(self, fields):
     @api.onchange('res_id')
     @api.multi
     def on_change_res_id(self):
+        if not self.res_id:
+            return {}
         mail_values = {}
-        if self.res_id and self._context.get('template_id'):
+        if self._context.get('template_id'):
```
Comment (rim-odoo) [inline addons/mail/wizard/email_template_preview.py]: @ppr-odoo Can you explain what are you trying to fix here?

Comment (ppr-odoo) [inline addons/mail/wizard/email_template_preview.py]: Here we decided to update the body value only if user select a actual record from selection, if user select blank value then simply don't update anything.Thank

**Comment by tde-banana-odoo** [comment]
Done at https://github.com/odoo/odoo/commit/e302ca7b3d67ead978022ecced51a6de4345983b . Thanks !

**Pull request closed without merging**